internal/git/git.go

```diff
@@ -184,17 +169,36 @@ func (g *Git) logDiff(aHash, bHash plumbing.Hash) error {
 }
 
 // BranchExist checks if the new branch exists
-func (g *Git) BranchExist(branchName string) (bool, error) {
-	_, err := g.repo.Reference(plumbing.ReferenceName(fmt.Sprintf("refs/remotes/origin/%s", branchName)), false)
```

Comment:
just want to confirm that this is still used elsewhere in the code; if not, a reminder to delete it :)

Reply:
Do you mean the `BranchExist` as a whole? In that case yes. It was moved to after the (potential) forking.

Reply:
Sorry - I meant the `plumbing.ReferenceName` method.

Reply:
Ah, that is a part of the go-git library :)